hopper.cabal

```diff
@@ -52,6 +52,19 @@ flag fullprofiling
   default: False
   manual: True
 
+executable hopperi
+  build-depends:
+    hopper,
+    base == 4.8.2.0,
+    lens >= 4.12 && < 4.14,
+    text >= 1.2 && < 1.3,
+    containers >= 0.5 && < 0.6,
+    vector >= 0.11 && < 0.12,
+    parsec,
+    haskeline
+  hs-source-dirs: hopperi
+  main-is: Main.hs
+
```

Comment:
for libs that are already mentioned / have constraints in the main hopper lib, its bad practice to pin the constraints further for an executable or test stanza